community/kernel/src/main/java/org/neo4j/kernel/builtinprocs/BuiltInProcedures.java

```diff
@@ -29,10 +29,10 @@
  */
 public class BuiltInProcedures
 {
-    public static void addTo( KernelAPI kernel ) throws ProcedureException
+    public static void addTo( Procedures procs ) throws ProcedureException
     {
-        kernel.registerProcedure( new ListLabelsProcedure( procedureName( "sys", "db", "labels" ) ) );
-        kernel.registerProcedure( new ListPropertyKeysProcedure( procedureName( "sys", "db", "propertyKeys" ) ) );
-        kernel.registerProcedure( new ListRelationshipTypesProcedure( procedureName( "sys", "db", "relationshipTypes" ) ) );
+        procs.register( new ListLabelsProcedure( procedureName( "sys", "db", "labels" ) ) );
```

Comment:
Mixing them in varargs is just a convenience shorthand for an internal utility. Keeping the name of the procedure and the implementation separate works I think - I guess either one is fine.